src/index.tsx

```diff
@@ -36,18 +36,21 @@ fetchConfig()
         setBaseUrl(config.API_HOST);
```

Comment:
setBaseUrl could be replaced by passing API_HOST directly into createServices function